code/datums/hud/human.dm

```diff
@@ -865,6 +855,18 @@
 					pos_x = 1
 					pos_y++
 
+		if (istype(master.loc,/obj/vehicle/)) //so we always see vehicle buttons
```

Comment:
DOUBT

Reply:
I was skeptical at first too, but, surprisingly, it works

Reply:
Or, at least, I wasn't able to get it to break despite extensive testing